crates/ra_ide_api/src/completion/complete_path.rs

```diff
@@ -73,7 +73,7 @@ mod tests {
     }
 
     #[test]
-    fn completes_mod_with_docs() {
+    fn complets_mod_with_docs() {
```

Comment:
wasn't `completes` the correct spelling?

Reply:
:sweat_smile: